script/deployment/timelocked/WhitelistPublishersTimelocked.s.sol

```diff
@@ -20,21 +20,20 @@ contract WhitelistPublishersTimelocked is Script {
     TimelockController public timelock;
 
     function run(bool isExecution) external {
-        isExecutionMode = isExecution;
         vm.startBroadcast();
-        execute();
+        execute(isExecution, true);
         vm.stopBroadcast();
     }
 
     // for testing purposes
     function run(address broadcastFrom, bool isExecution) public {
-        isExecutionMode = isExecution;
         vm.startBroadcast(broadcastFrom);
-        execute();
+        execute(isExecution, true);
         vm.stopBroadcast();
     }
 
-    function execute() public {
+    function execute(bool isExecution, bool send) public returns (bytes memory) {
```

Comment:
what is the use cases to pass
execute(true, false)
execute(false, true)
execute(true, true)
execute(false, false)
?

Reply:
The first parameter `isExecution` defines if the transaction needs to be scheduled or it should execute scheduled transaction. The second parameter `send` defines if timelock should be called in this function or it just returns calldata. The first option is used when timelock permissioned address is EOA, the second option is for multisig.